client/webpack.config.js

```diff
@@ -392,6 +393,37 @@ const webpackConfig = {
 		new ExtensiveLodashReplacementPlugin(),
 	].filter( Boolean ),
 	externals: [ 'keytar' ],
+	...( shouldUsePersistentCache
+		? {
+				cache: {
+					// eslint-disable-next-line inclusive-language/use-inclusive-words
+					// More info in https://github.com/webpack/changelog-v5/blob/master/guides/persistent-caching.md
+					type: 'filesystem',
+					version: [
+						shouldBuildChunksMap,
```

Comment:
Just to confirm, we're fine with having `true` or `false` as pieces of the cache key here, right?

Reply:
Yes, they will be serialized as `-false-` in the cache key.

That being said, I'm not 100% on how anything cache related works, there is virtually no docs or examples online. That's why I left the `infrastructureLogging: { debug: /webpack\.cache/}`, I hope more logging will help us understand the caching better.